crates/autopilot/src/domain/settlement/mod.rs

```diff
@@ -32,6 +32,15 @@ impl Settlement {
     /// id.
     const META_DATA_LEN: usize = 8;
 
+    pub fn native_surplus(&self, prices: &auction::Prices) -> Result<eth::Ether, trade::Error> {
+        self.trades
+            .iter()
+            .map(|trade| trade.native_surplus(prices))
+            .try_fold(num::Zero::zero(), |acc, score| {
+                score.map(|score| acc + score)
+            })
+    }
```

Comment:
can't this just become `sum` given that we implement `std::ops::Add`?

Reply:
Implemented `sum` for `eth::Ether` in autopilot and driver.